packages/jaeger-ui-components/src/TraceTimelineViewer/TimelineHeaderRow/TimelineColumnResizer.test.tsx

```diff
@@ -32,74 +28,26 @@ describe('<TimelineColumnResizer>', () => {
 
   beforeEach(() => {
     props.onChange.mockReset();
-    wrapper = mount(<TimelineColumnResizer {...(props as unknown as TimelineColumnResizerProps)} />);
-    instance = wrapper.instance();
+    render(<TimelineColumnResizer {...(props as unknown as TimelineColumnResizerProps)} />);
```

Comment:
if you add a mock `columnResizeHandleHeight` to the props, you can type them correctly `const props: TimelineColumnResizerProps = { columnResizeHandleHeight: 10, ... }` and avoid this type cast

Reply:
Done! https://github.com/grafana/grafana/pull/61156/commits/ef5419070223c455127d003d291b98cca1abddb9